### PR TITLE
Allows to set a custom health check path

### PR DESCRIPTION
**Pull request by jcaromiq**
With this change, we can customize the health check path, 
By default is working as now, with the /.well-known/apollo/server-health

### CODE REVIEW

File: docs/source/configuration/health-checks.mdx [hunk 1]
```diff
@@ -6,3 +6,8 @@ description: Determining the router's status
 Health checks are often used by load balancers to determine whether a server is available and ready to start serving traffic.
 
 The Apollo Router supports a basic HTTP-level health check. This is enabled by default and is served at the URL path `/.well-known/apollo/server-health`. This returns a `200` status code if the HTTP server is successfully serving. It does not invoke any GraphQL execution machinery.
+You can change this path by setting `server.health_check_path`:
+```yml
```
Comment (bnjjj) [inline docs/source/configuration/health-checks.mdx:10]:
```suggestion
```yaml title="router.yaml"
```
Our unit tests detect this to automatically test our examples in documentation

**Review by bnjjj — CHANGES_REQUESTED** [review]
Thanks for your contribution, it looks good overall. Just a small suggestion

**Comment by bnjjj** [comment]
Could you also add a changelog entry in `NEXT_CHANGELOG.md` please ?

**Comment by bnjjj** [comment]
@jcaromiq did you sign the CLA ? I'm just wondering if the bot didn't already update the status or if you didn't already signed the CLA.

**Comment by jcaromiq** [comment]
> @jcaromiq did you sign the CLA ? I'm just wondering if the bot didn't already update the status or if you didn't already signed the CLA.

Hello @bnjjj, yes I signed the CLA 2 days ago with my github account

**Comment by bnjjj** [comment]
Hmm weird, could you try to `git commit --amend` and force push to try to retrigger the CLA bot ? I'm sorry for this.

**Review by bnjjj — APPROVED** [review]

**Review by Geal — APPROVED** [review]

File: NEXT_CHANGELOG.md [hunk 2]
```diff
@@ -25,3 +25,12 @@ By [@USERNAME](https://github.com/USERNAME) in https://github.com/apollographql/
...
 # [0.9.4] (unreleased) - 2022-mm-dd
+
+## 🚀 Features
+### Allow to set a custom health check path ([PR #1164](https://github.com/apollographql/router/pull/1164))
+Added the possibility to set a custom health check path
+  ```yaml
+server:
+  # Default is /.well-known/apollo/server-health
+  health_check_path: /health
+```
```
Comment (bnjjj) [inline NEXT_CHANGELOG.md:36]:
```suggestion
```

By [@jcaromiq](https://github.com/jcaromiq) in https://github.com/apollographql/router/pull/1164
```

**Pull request merged**